### ISSUE TITLE
Script vs container

### ISSUE THREAD

**Issue by errordeveloper** [chuckbutler/layer-scope#2]
There is no major need for using the script, especially if has to be checked in, and it's been modified (5ead1205a4ec463aee9d26eb7374508ce032c094). Running container image directly should be sufficient.

**Comment by errordeveloper** [comment]
```
docker run \
  --privileged=true \
  --net=host \
  --pid=host \
  -p 0.0.0.0:4040:4040 \
  -v /var/run/:/var/run:rw \
  --probe.docker true
```

**Comment by lazypower** [comment]
Thats awesome. I was using the script as it was the recommended path to get started that i found in the docs.

I'll create a feature branch for this and update it with the newer CLI wrappers we're baking into the docker layer so its nice and concise.

**Comment by errordeveloper** [comment]
Great, I'm looking forward to review it :+1: Just for the record, the script is meant to be the official way when you wanna run _latest_, i.e. flags might come and go, but that's not an issue if you pin the version.

**Comment by lazypower** [comment]
@errordeveloper sorry for the delay in handling this. I did however strip that older script, leaving the behavior of docker-compose to drive the deployment of just the container.

Admins can now version lock the container via the 'version' setting in config.yaml

I believe this closes out #2 and introduces the CLI wrappers i was referring to :smiley:

**Comment by lazypower** [comment]
Closed with #3
